Updates from the community

dsmorse opened the pull request.

AWhetter committed:
docs(gridster): fixed a typo in Gridster class documentation

Commit:
fix(gridster): changed "instanceof jQuery" to "instanceof $"

Original code will break if jquery noconflict is applied, where $ and jQuery produces jQuery instance of different versions.

rept committed:
fix(gridster): wrong addition solved in add_faux_rows/cols by adding parseInt

Closes #426, #425

joaoafrmartins committed:
feat(gridster): Common.js support

Closes #434

joaoafrmartins committed:
commonjs

vieron committed:
fix(draggable): namespace events with unique ids

Until now if you had multiple instances of Draggable in the same page,
when destroying one of them, all of the events binded to other
instances were also destroyed.

So, we are namespacing event names with unique id’s per instance to ensure
only that only the corresponding events are unbinded.

The other solution is to cache references to the event handlers, but most
of them are proxied or throttled which makes more mess...

vieron committed:
Release v0.5.6

yfix committed:
Merge pull request #1 from ducksboard/master

Update from origin

Commit:
Fix gridster destroy

Removes the coords data from each element.

Commit:
Update jquery.gridster.js

Commit:
Added ability to specify scroll container for drag/resize

Commit:
fix issue #469

remove trailing comma to be more IE compliant

Commit:
fix (Gridster.js):  destroy  resize_api

Call  resize_api.destroy()  to fix  memory leak.

Commit:
fix(gridster): destroy resize_api

Call  `resize_api.destroy()`  to fix  memory leak.

Closes #473

Commit:
Added destroy of widget coords and resize api

sogawa committed:
rename test files.

sogawa committed:
add element "#qunit-testresult"

ericmann committed:
Accepting Resize API pull request. @props tormoalto

sogawa committed:
Convert to integer type of property value when sorting.

ericmann committed:
Cache $(window) and $(this) selectors for performance

ericmann committed:
Fix the memory leak that was incorrectly iterating rows and columns

Commit:
add model_id parameter to add_widget method in order to create a relationship between widget and backbone model

Commit:
re-name model_id parameter

veeamGMBH committed:
delete jquery dependency

Commit:
Fix spelling mistake

ericmann committed:
Fix Widget Right Edge Collission Detection

Apply the fix from duckboard's issue tracker. Fixes #2.

ericmann committed:
Grunt

ericmann committed:
Bumping to v0.5.7

Commit:
fix(gridster): changed the way widgets were getting positioned so that margins are actually the same size that the user specified all the way around the grid

feat(gridster): added responsive nature to grid

Commit:
Release v0.6.0

Commit:
fix(gridster): fixing drag limit issues when using autogrow_cols

Commit:
fix(gridster): fixed expand_widget bug not expanding full width of window
fix(gridster): user can specify size_x when expanding widget

Commit:
Release v0.6.1

Commit:
fix(gridster): forcing height of gridster container to auto when in collapsed mode

Commit:
Release v0.6.2

Commit:
fix(gridster): fixes bug where widgets would overlay other widgets after a resize

fhawkes committed:
Merge pull request #2 from DecksterTeam/resize-overlap

fix(gridster): fixes bug where widgets would overlay other widgets after...

Commit:
fix(gridster): adding bower.json file

Commit:
Merge branch 'master' of github.com:DecksterTeam/gridster.js

Commit:
fix(gridster): adding bower.json file

Commit:
Fix for bug #493 including unit tests to prove.

Run npm install and visit test/jquery.gridster-amd.html in browser.

Commit:
refactor(gridster): adding callback to add_widget function

Commit:
fix(gridster): fixing resize limits when in fixed width mode
feature(gridster): added fix_to_content function to fit widget dimensions to a given width and height
feature(gridster): added resize_widget_dimensions function
refactor(gridster): moved code to set number of columns in grid to helper function

Commit:
Release v0.6.3

ericmann committed:
Add an optional final parameter to add_widget.

ericmann committed:
Grunt changes

ericmann committed:
Merge in fade changes

diogobenica committed:
Change jQuery reference for $.

Commit:
Include move_widget function (Move a widget to a specific row and column.).

Commit:
add function move_widget in dist/jquery.gridster.js

Commit:
#3 Adds ability to automatically center widgets in the grid.

Commit:
Release v0.6.4

Commit:
#3 Adds ability to automatically center widgets in the grid.

Commit:
Move the check for responsive width above the calculations of width.  This will prevent the calculated values changing the width for browsers which don't respect min-width

Commit:
Move the check for responsive width above the calculations of width.  This will prevent the calculated values changing the width for browsers which don't respect min-width

Commit:
fix(gridster): fixed bugs in centering_widgets (widgets were getting smushed when being resized) and fixed bug with min_width

Commit:
Release v0.6.5

Commit:
Release v0.6.5

Commit:
chore(grunt): fixing versioning system

Commit:
Merge branch 'master' of https://github.com/DecksterTeam/gridster.js

Commit:
fix a NAN error that could occur if using center_widget if the grid was in responsive mode.
Error check the shift cols, to ensure they can't move left past col 1
make a change from 100% to 100VW for better browser support of min-width

Commit:
Merge from abril/gridster.js

* merging:
  add function move_widget in dist/jquery.gridster.js
  Include move_widget function (Move a widget to a specific row and column.).
  Change jQuery reference for $.

Commit:
Merge remote-tracking branch 'eric/master'

  Grunt changes
  Add an optional final parameter to add_widget.
  Bumping to v0.5.7
  Grunt
  Fix Widget Right Edge Collission Detection
  re-name model_id parameter
  add model_id parameter to add_widget method in order to create a relationship between widget and backbone model
  Fix the memory leak that was incorrectly iterating rows and columns
  Cache $(window) and $(this) selectors for performance
  fix (Gridster.js):  destroy  resize_api

Commit:
Merge remote-tracking branch 'ducksboard/bug-493-jquery-no-global-fail'

* ducksboard/bug-493-jquery-no-global-fail:
  Fix for bug #493 including unit tests to prove.

Commit:
Merge remote-tracking branch 'Roger-Waters/gridster.js'

*   Fix spelling mistake

Commit:
Merge remote-tracking branch 'sogawa/gridster.js/strict-sort'

* sogawa/gridster.js/strict-sort:
  Convert to integer type of property value when sorting.
  add element "#qunit-testresult"
  rename test files.

Commit:
Merge remote-tracking branch 'bfarka/gridster.js/fix-trailing-comma'

* bfarka/gridster.js/fix-trailing-comma:
  fix issue #469

Commit:
Merge remote-tracking branch 'mooglemoogle/gridster.js/master'

* mooglemoogle/gridster.js/master:
  Added destroy of widget coords and resize api
  Added ability to specify scroll container for drag/resize

Commit:
Merge remote-tracking branch 'Toosick/gridster.js/patch-1'

* Toosick/gridster.js/patch-1:
  Update jquery.gridster.js
  Fix gridster destroy

Commit:
Merge remote-tracking branch 'joaoafrmartins/gridster.js/master'

* joaoafrmartins/gridster.js/master:
  commonjs
  support UMD

Commit:
Merge remote-tracking branch 'rept/gridster.js/addition_problem_add_faux_rows'

* rept/gridster.js/addition_problem_add_faux_rows:
  Also added parseInt to the add_faux_cols method
  Also added parseInt to the add_faux_cols method
  Wrong addition solved in add_faux_rows by adding parseInt

Commit:
Merge remote-tracking branch 'aegisrunestone/gridster.js/master'

* aegisrunestone/gridster.js/master:
  Changed "instanceof jQuery" to "instanceof $"

Commit:
Merge remote-tracking branch 'Tropicalista/gridster.js/master'

Commit:
Merge remote-tracking branch 'dragonier/gridster.js/master'

* dragonier/gridster.js/master:
  add_resize_handle: multiple span (resizeable) entries in widgets

Commit:
Merge remote-tracking branch 'AWhetter/gridster.js/fix_doc_typo'

* AWhetter/gridster.js/fix_doc_typo:
  Fixed a typo in Gridster class documentation

Commit:
Merge remote-tracking branch 'cpascu/gridster.js/faux-rows-patch'

* cpascu/gridster.js/faux-rows-patch:
  Fixed typeof comparison to match against string.
  Fixed issue where existing widget position would be overwritten on gridmap when adding a faux cell to accommodate another widget.

Commit:
Merge remote-tracking branch 'dasmall/gridster.js.git/master'

* dasmall/gridster.js.git/master:
  Removing previously added style tags before adding new one. Hopefully fixes #211 and #294.

Commit:
Merge remote-tracking branch 'AdrianTeh/gridster.js.git/master'

* AdrianTeh/gridster.js.git/master:
  fixes-add-widget-overlapping-issue-#340

Commit:
Merge remote-tracking branch 'kwak27/gridster.js/fix_add_widget'

* kwak27/gridster.js/fix_add_widget:
  fix add_widget to use correct size_y when adding rows

Commit:
Merge remote-tracking branch 'andreas-trad/gridster.js.git/patch-1'

* andreas-trad/gridster.js.git/patch-1:
  Prevent multi <style> generation

Commit:
Merge remote-tracking branch 'rebugger/gridster.js.git/resize-widget-dimensions'

* rebugger/gridster.js.git/resize-widget-dimensions:
  fix min-width and min-height after resize
  add resize_widget_dimension() resize widgets after initialization
  fix min-width and min-height after resize
  add resize_widget_dimension() resize widgets after initialization

Commit:
Merge remote-tracking branch 'yfix/gridster.js.git/master'

* yfix/gridster.js.git/master:

Commit:
Merge remote-tracking branch 'Laboratory/gridster.js.git/master'

* Laboratory/gridster.js.git/master:
  delete jquery dependency

Commit:
Update Readme

Commit:
Release v0.6.6

Commit:
Fix my merge error from the merge from remote-tracking branch 'cpascu/gridster.js/faux-rows-patch'

Commit:
Merge from dustmoo/gridster.js

* commit '262d115cf8b5bdaed40a7e5b6aaa6df6e71b6a07':
  Added swaping queue, better handling of large to small swaps
  Added swaping queue, better handling of large to small swaps
  More predictable default behavior
  Updated Readme
  Added property to prevent larger widget shifting
  Added grid-swap behavior

Commit:
Merge from dustmoo/gridster.js

* commit '8f431a423302c7c58f511d5ede9270077993f9b3':
  Updated Readme
  Updated Readme
  Version Updates
  Serialize bug fix
  Added max_rows and drag restriction - ready for integration :)
  Removed console.logs
  Removed console.logs
  Good enough swap logic for now
  Swap queue awesomeness
  Swap queue awesomeness
  Fix row issue
  working on map grid functionality

Commit:
Merge commit '6eb521015cda5212f6fa92492dbe4050bd92e00c'

* commit '6eb521015cda5212f6fa92492dbe4050bd92e00c':
  Update Readme
  Updated sample
  Better handling of widgets slideing down
  update readme
  rebuilt dist
  Added Swap Demo
  Added disabled dragging init feature gridster.js
  Working on Static widgets

Commit:
Update Samples

Commit:
bring the Demo's into  master for testing & Automate the updates to gh-pages branch

Commit:
Add unit testing for core API, ensure all demos work, and add unit testing to ensure they stay working.

vieron committed:
feat(draggable): autoscrolling

If the element being dragged is inside a scrollable container, keeping the
mouse in one of the sides of the container will trigger scroll.

vieron committed:
fix(gridster): ensure widget dimensions and coords are always ints

Thanks to @sogawa for reporting

vieron committed:
chore: bump up jquery version to 2.1.3

vieron committed:
feat(gridster): browserify compatibility

Commit:
fix(gridster): preventing gridster from adding extra resize handles

Commit:
Revert "fix(gridster): preventing gridster from adding extra resize handles"

This reverts commit 9d077da676826606243c2552dc9997c492687203.

Commit:
preventing gridster from creating extra resize handles while centering

fhawkes committed:
Merge pull request #6 from DecksterTeam/dadams

Resize handles

Commit:
Release v0.6.6

vieron committed:
tests(gridster): add basic test suite for gridster

Tests are build with mocha + chai, and you can run them from the command line with `grunt karma` or opening `test/index.html` in your browser.

This is a first approach, there are hundreds of tests more that should be added.
I’ve ported some tests on PRs but more needs to be done.

vieron committed:
chore(gridster): remove unused code

vieron committed:
feat(gridster.css): remove possible default pading

vieron committed:
feat(gridster): add config to set custom show/hide widget methods

by default jQuery’s `hide` and `show` methods are used. You could also
use fadeIn/fadeOut or write your own kind of jQuery plugin like
`$.fn.showInAFancyWay` and use `showInAFancyWay` as the value in the
show_method config option.

If you want to keep the previos behaviour, you need to set
`hide_method` option to `’fadeOut’`

Breaking Changes

`remove_widget` and `remove_all_widgets` methods not return a promise
instead of the gridster instance

vieron committed:
fix(gridster): `resize_widget` also accepts HTMLElements

for consistency with the `remove_widget`

vieron committed:
fix(gridster): ensure coords instances are destroyed on widgets

…when removing a widget, or when destroying the gridster instance.
Before this change, if you called destroy without `remove` set to true,
coords instances was leaking jQuery data objects.

Also nullified the gridmap, and faux_grid arrays.

vieron committed:
style(jshint): add jshint config and setup grunt task

vieron committed:
style(jshint): adapt code to pass jshint

Commit:
rev jquery version

Commit:
0.6.7 release

fhawkes committed:
fix(gridster): responsive width now resizes based off wrapper not window

Commit:
update dependencies and jshint

Commit:
Merge commit 'e69c3e8f64aa4557ef032e4d0d8185e83b1aed21'
fix(gridster): responsive width now resizes based off wrapper not window

Commit:
Release v0.6.8

ericop committed:
typo on "the" in "inactivity over the last year"

Commit:
Merge pull request #4 from ericop/patch-1

typo on "the" in "inactivity over the last year"

Commit:
Fix bug with responsive layout if grid in wrapped in a container which has padding. It now limits the width of the widget to the containing element (also helps with scrollbars)

Commit:
Add new demo page and allow two new features based on Issue #6:
- delay moving components until drag is complete
- allow exact position of grid components (do not move up)

Commit:
Remove debug code

Commit:
Fix gridster width setting in responsive mode

* needs to use 100% instead of 100vw because the container may be smaller than the viewport width

Commit:
Fix grid calculations when wrapper/gridster element is not centered

Commit:
lint fix, generated dist

Commit:
fix initialization of sticky layout.

Commit:
Merge branch 'zeroedin-bill-master'

Commit:
Fix issue where dragging a small widget up onto a bottom corner of another widget led to persistent overlap

Commit:
Fix undefined var reference to this.shift_widgets_up - should ref option

Commit:
Fix undefined exception in update_widgets_dimensions

Commit:
fix unit tests

Commit:
Merge branch 'master' of git://github.com/zeroedin-bill/gridster.js into zeroedin-bill-master

Commit:
Fix expanding widget issue (allow it to collapse again)

Commit:
JS Doc changes

Commit:
Various bug fixes for moving widgets around

Commit:
add unit test to build process

plamoureux committed:
jspm config

plamoureux committed:
jspm config

plamoureux committed:
jspm config

plamoureux committed:
jspm config

plamoureux committed:
jspm config

plamoureux committed:
jspm config

Commit:
Don't ignore src, I need it to build with webpack.

Commit:
Fix invalid require('./jquery-gridster.js')

Commit:
Remove if statement from register_widgets that cause them to be dropped from the DOM if they don't have a specific class on them.  Fixes bug report #9

Commit:
Merge pull request #11 from zeroedin-bill/master

for traditional bower users like myself, its just one more thing to 'preen', but if it helps you lets make it happen

ened committed:
Create a Ruby on Rails gem

Commit:
enable travis

Commit:
travis config for bower

Commit:
install bower & grunt globally for travis

Commit:
add badge for gem

Commit:
Release v0.6.9

Commit:
fixed highlighting while resize is true

Commit:
update ruby build config

Commit:
update readme

ened committed:
RoR gem missed the Engine files

ened committed:
Fixed `npm install` warning

npm WARN package.json gridster@0.6.9 homepage field must start with a protocol.

Commit:
Merge pull request #14 from ened/fix/package-json-url

Fixed `npm install` warning

Commit:
Merge pull request #15 from ened/fix/rails-gem

RoR gem missed the Engine files

Commit:
Merge branch 'master' of https://github.com/dsmorse/gridster.js
updates to make ruby on rails gem work. see pull Request #14 & #15

Commit:
chore: update RoR build process

Prep for 0.6.10 release.

Commit:
make npm ready

Commit:
chore: Merge remote-tracking branch 'spalger/master' - make npm ready

Commit:
chore: Merge remote-tracking branch 'josephndenton/master'

fixed highlighting while resize is true

Commit:
chore: Merge remote-tracking branch 'mergeSource/master'

Add jspm support

Commit:
chore: convert from css to Less

Commit:
feature: Add Chaos demo to test widget placement.

Commit:
fix: convert from JQury to DOM

Move from Jquery based width() for calculations to dom calls to avoid conflicts with webcomponents.js polyfills

Commit:
fix(gridster): Add additiona error checking on widgets removal

…when remove_widget is called with an empty element we need to abandon the action early to avoid exceptions

Commit:
Extended default options by two functions for displaying and hiding widgets so the default behavior can be overridden

Commit:
update jsdoc

Commit:
update generated files

Commit:
update generated files

Commit:
Revert "update generated files"

This reverts commit 6c2e2d4099ddc666cf50bf21f5f5a5d7cc5728bc.

Commit:
update generated files (2)

Commit:
Merge pull request #19 from heartdisease/master

feat:  Extended default options by two custom functions for showing and hiding widgets

Nedlinin committed:
Fix issue with the dragging of widgets pushing off to the right funny with larger margin sizes.  The margin was not being taken into account for the new position properly.  This meant that if you were in column 10 and the margin was set to 20px the previous calculation was off by 200 pixels.  New calculation for the placeholder position and the drag stop position takes the margin for the sizes multiplied by the column we want to determine how far to offset left.

Nedlinin committed:
Build new distributable files.

Commit:
Merge pull request #22 from Nedlinin/master

Fix issue with the dragging of widgets pushing off to the right funny…

randoum committed:
Remove 'bootstrap-typeahead-rails' require

Commit:
Merge pull request #23 from randoum/patch-1

Remove 'bootstrap-typeahead-rails' require statement

emackey committed:
Fix the responsive grid to be able to shrink.

stenverbois committed:
fix: Apply fix from #22 to vertical positioning

Commit:
Merge pull request #26 from emackey/fix-responsive-grid

Fix the responsive grid to be able to shrink.

Commit:
Merge pull request #27 from stenverbois/master

fix: Apply fix from #22 to vertical positioning

luke-browning committed:
Fire draggable.stop event after data attributes have been updated.

Commit:
Merge pull request #31 from luke-browning/master

Fire draggable.stop event after data attributes have been updated.

Commit:
fix(fn.generate_stylesheet): only return new stylesheet if colWidth and
this.$wrapper.width() are > 0

With reponsive layouts, on window resize, colWidth and
this.$wrapper.width() can be <= 0. This causes widget widths to be set
as <= 0. I think it is fair to assume that both the colWidth and
$wrapper.width() must be greater than 0 in order to generate a valid
stylesheet.

alexandernst committed:
Fix min width/height per widget not being calculated

The min width/height per widget isn't getting caalculated, which wrongly calculates the total height oh the gridster itself when resized. This patch fixes this.

As a bonus, I removed a duplicated function which was doing half the things this function is doing.

Commit:
Merge pull request #35 from ZachLiss/fix-stylesheet-generation

fix(fn.generate_stylesheet): only return with valid widths

Commit:
Fix issue where resizing a widget at the bottom of the screen can cause the screen to scroll up and mess up the resize.

This was happening because of height recalculation while the widget is not in the gridmap. This adds a variable to track whether a resize is occurring, and if so, defer height recalculation in the critical section.

Commit:
Merge pull request #38 from zeroedin-bill/master

Fix issues with resizing a widget at the bottom of the screen

Commit:
Merge pull request #36 from alexandernst/master

Fix min width/height per widget not being calculated

stephwag committed:
add flag to make widgets only move down

Commit:
Merge pull request #39 from stephwag/master

Add flag to make widgets only move down

MM0nter0 committed:
fix bug with sticky position widgets

Widget is removed from gridmap on_stop_drag only if changes its position
Overlapped widget is moved below the new position, taking into account the size of the widgets.

Closes #30

Commit:
Merge pull request #43 from M0nter0/master

fix bug with sticky position widgets

Commit:
Prep release of 0.7.0

DyckGerman committed:
Improved processing of cells that player initially occupies

Commit:
Merge pull request #64 from DyckGerman/occupied-cells-processing-fix

Improved processing of cells that player initially occupies

Commit:
Added to NPM, made name NPM friendly in package, added instructions

pboucher1960 committed:
Fix bug that shift widgets up when moving or resizing an upper widget even when options.shift_widgets_up is false #72

fix(gridster.js): Option shift_widgets_up=false is not always
implemented.

Add a validation on gridster.options.shift_widgets_up in
gridster.fn.mutate_widget_in_gridmap before calling remove_empty_cells.
Call remove_empty_cells only when gridster.options.shift_widgets_up is
set to true.

Closes #72

Commit:
Merge pull request #73 from pboucher1960/issue_72_shift_widgets_up

Fix bug that shift widgets up on move or resize when options.shift_widgets_up is false

Commit:
Fix (gridster.js): Option collision.wait_for_mouseup: true and shift_widgets_up: false not working properly on collisions

Remove a condition that tests we are using a widget and a player to force update occupied cells.

Closes #75

Commit:
Merge pull request #76 from anemmas/master

Fix (gridster.js): Option collision.wait_for_mouseup: true and shift_…

Commit:
Merge pull request #71 from baconface/master

Added to NPM, made name NPM friendly in package, added instructions

Commit:
Add ignore_self_occupied option to allow gridster objects to be moved single columns/rows regardless of the size of the object itself.

Commit:
Merge pull request #78 from Gfriedman21/master

Add ignore_self_occupied option to allow gridster objects to be moved…

dennisvg111 committed:
feat($browser): get_offset
-changed options.limit to options.limit.width and height
-use limit: {width: true, height: true} to set when creating a new grid, default is still width true, height false

dennisvg111 committed:
fix($browser): fn.on_start_drag
-fix a bug where this.set_dom_grid_height doesn't have a max height check first

dennisvg111 committed:
fix:($browser): fn.on_stop_drag inside widgets on max row
-fix a bug where widgets on max row get pushed further when another widget stops on them
-the moved widget now moves back instead

dennisvg111 committed:
fix($compile) added missing semicolons

dennisvg111 committed:
fix($browser): make fn.move_widget_down return a success bool, to prevent towers of widgets dropping down too far
-on_stop_drag now checks the return value rather than checking one row below

dennisvg111 committed:
fix($compile): semicolons, again
-semicolon

dennisvg111 committed:
fix($browser): move_widget_down
-movedown now checks for height as well to know if a widget can move down

dennisvg111 committed:
feat($browser): add_widget failure
-if a new widget doesn't fit, don't add it to the list and return false

Commit:
Merge pull request #81 from dennisvg111/draggablefixes

fixes to bugs to do with maxrow, adding a row in on_start_drag, and adding the option to limit both width and height

dustmoo commented:
Thanks for all your work on this @dsmorse, I'm going to set aside some time in the coming week to see if I can get these changes merged into this long neglected branch.

dsmorse commented:
that will be awesome... a lot of people have been giving me PR's, so my branch is even further ahead than this, I'd love to see my branch go away it it go back to you.

dennisvg111 committed:
fix($compiling) added support for people using the deprecated limit: bool
-added support + a warning for people using the old limit
-fixed a small bug in fn.resizable

dennisvg111 committed:
changed deprecated warning message to use single quotes

Commit:
Merge pull request #82 from dennisvg111/draggablefixes

fix($compiling) added support for people using the deprecated limit: …

dennisvg111 committed:
$fix(build) update gruntfile so changing packagename won't break build

dennisvg111 committed:
$fix(build) build changes in src to dist

dennisvg111 committed:
$fix(build) fix css build when package name is not gridster

dennisvg111 committed:
$feat(browser) re-add ignore_self_occupied option
- was directly added to dist files, and thus removed when building

dennisvg111 committed:
$feat(build) readded ignore_self_occupied to dist build

Commit:
Merge pull request #88 from dennisvg111/update_dist_build

Update dist build, fixed gruntfile to allow other package names

dennisvg111 committed:
$fix(browser) fixed a minor issue with fn.set_limits
- preview displacement was caused by wrongfully resized widgets,
- * this.options.distance wasn't actually needed

dennisvg111 committed:
$fix(build) changes build to dist version

dennisvg111 committed:
$feat(browser) added a function to change the numbers of rows
- allows people to easily set a new number of rows when a limit is desired, but the limit is not known before (responsiveness)

benjamindeclercq committed:
fn.resizable: limit

benjamindeclercq committed:
Revert "fn.resizable: limit"

This reverts commit ef495c8da21fae2321bf38184e9c7e5e2139565e.

benjamindeclercq committed:
fn.resizable: limit

Commit:
Merge pull request #91 from BenjaminDeClercq/master

fn.resizable: limit

Commit:
Merge pull request #90 from dennisvg111/fix_preview_maxleft

Fix preview maxleft, add set_num_rows function

detain committed:
updates allowing gridster to work with jquery 2.x and 3.x

detain committed:
updates forcing testing of both jquery 2 and 3

dsmorse committed:
Merge pull request #100 from detain/master

add support for jQuery 3.x too

Commit:
fix bug

alexnb committed:
fix: resizing an element from very large to very small was not possible because the resulting size deviated from the mouse position, especially in case of small grid dimensions

alexnb committed:
version 0.7.1

alexnb committed:
version 0.7.1

progitter committed:
Update Gruntfile.js

leithhobson committed:
Merge pull request #103 from Mrjeff578575/master

Widget doesn't  auto move up when remove widget

fixmycode committed:
Updates the grunt-contrib-qunit package to the latest version

That way, phantomjs is updated and the test passes on travis without any other changes.

alexnb committed:
Revert "version 0.7.1"

This reverts commit 1c84797db4bbf55da785c97018a603f60fd8b4e1.

(as asked by @fixmycode for PR #112)

alexnb committed:
Revert "version 0.7.1"

This reverts commit 8639a2915a5bd9df26473692dea383d46ae653f6.

(as asked by @fixmycode for PR #112)

mikegoodstadt committed:
Invalid domain gridster.net

Remove link to empty domain

Commit:
Update README for 0.8.0 gem instructions; update package.json to 0.8.0; update dist contents based on current package & Gruntfile

Commit:
Update README for 0.8.0 gem instructions; update package.json to 0.8.0; update dist contents based on current package & Gruntfile

Commit:
Merge branch 'master' of github.com:kbroderick/gridster.js

leithhobson committed:
Merge pull request #130 from kbroderick/master

Address #129 -- README instructions for Gemfile

leithhobson committed:
Merge pull request #120 from alexnb/modified-pr-112

Resizing an element from very large to very small was not possible (corrected)
